commit 1f51fdfead03cdc30df1fed6ed203907643b8d0d
Author: Todd Fujinaka <todd.fujinaka@intel.com>
Date:   Fri Mar 17 00:48:19 2017 +0000

    igb: add i211 to i210 PHY workaround
    
    [ Upstream commit 5bc8c230e2a993b49244f9457499f17283da9ec7 ]
    
    i210 and i211 share the same PHY but have different PCI IDs. Don't
    forget i211 for any i210 workarounds.
    
    Signed-off-by: Todd Fujinaka <todd.fujinaka@intel.com>
    Tested-by: Aaron Brown <aaron.f.brown@intel.com>
    Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linux-4.9.47/drivers/net/ethernet/intel/igb/e1000_phy.c
===================================================================
--- linux-4.9.47.orig/drivers/net/ethernet/intel/igb/e1000_phy.c
+++ linux-4.9.47/drivers/net/ethernet/intel/igb/e1000_phy.c
@@ -82,7 +82,7 @@ s32 igb_get_phy_id(struct e1000_hw *hw)
 		phy->ops.write_reg(hw, I347AT4_PAGE_SELECT, 0);
 
 	/* ensure PHY page selection to fix misconfigured i210 */
-	if (hw->mac.type == e1000_i210)
+	if ((hw->mac.type == e1000_i210) || (hw->mac.type == e1000_i211))
 		phy->ops.write_reg(hw, I347AT4_PAGE_SELECT, 0);
 
 	ret_val = phy->ops.read_reg(hw, PHY_ID1, &phy_id);
Index: linux-4.9.47/dummy/rpi_1415_1f51fdfead03cdc30df1fed6ed203907643b8d0d.txt
===================================================================
--- /dev/null
+++ linux-4.9.47/dummy/rpi_1415_1f51fdfead03cdc30df1fed6ed203907643b8d0d.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
